DatadogCore/Sources/Datadog.swift

```diff
@@ -301,6 +301,14 @@ public enum Datadog {
         core?.clearAllData()
     }
 
+    /// Stops the initialized SDK instance attached to the given name.
```

Comment:
Would be great to get a short documentation stating what happens to the data processed/stored by this instance.